packages/vm/Cargo.toml

```diff
@@ -45,9 +45,9 @@ thiserror = "1.0.26"
 wasmer = { version = "=4.2.5", default-features = false, features = ["cranelift", "singlepass"] }
 wasmer-middlewares = "=4.2.5"
 strum = { version = "0.25.0", default-features = false, features = ["derive"] }
-# For heap profiling. Only used in "memory" example. This has to be a non-dev dependency
+# For heap profiling. Only used in "heap_profiling" example. This has to be a non-dev dependency
```

Comment:
This should also be changed for the `dhat-heap` feature above.

Reply:
done, thanks